### PR TITLE
notifications template

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/src/Notifications/AdvsModuleNotificationsTemplateSeeder.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+class AdvsModuleNotificationsTemplateSeeder extends Seeder
+{
+    public function run()
+    {
+        if (is_module_installed('visiosoft.module.notifications')) {
+
+            $templates = [
+                [
+                    'en' => [
+                        'message' => '<p><strong>&nbsp;Your post <a href="{url}" target="_blank">{name}</a> has been created successfully.</strong></p>',
```
Comment (fatihalp) [inline addons/default/visiosoft/advs-module/src/Notifications/AdvsModuleNotificationsTemplateSeeder.php:17]: &nbsp; ?

File: addons/default/visiosoft/profile-module/src/Notifications/ProfileModuleNotificationsTemplateSeeder.php [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                                                             </td>
+                                                         </tr>
+                                                         <tr>
+                                                             <td class="esd-block-text es-p20t" align="left">
+                                                                 <p style="color: #707070; font-size: 16px;">Hi user,</p>
+                                                             </td>
+                                                         </tr>
+                                                         <tr>
+                                                             <td class="esd-block-text es-p15t" align="left">
+                                                                 <p style="color: #707070; font-size: 16px;">Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
```
Comment (fatihalp) [inline addons/default/visiosoft/profile-module/src/Notifications/ProfileModuleNotificationsTemplateSeeder.php:47]: Please remove all lorem ipsum

**Review by fatihalp — CHANGES_REQUESTED** [review]
delete lorem ipsum

File: addons/default/visiosoft/profile-module/src/Notifications/ProfileModuleNotificationsTemplateSeeder.php [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+            ];
+
+            foreach ($templates as $template) {
+                $this->createTemplate($template);
+            }
+        }
+    }
+
+    public function createTemplate($params)
```
Comment (fatihalp) [inline addons/default/visiosoft/profile-module/src/Notifications/ProfileModuleNotificationsTemplateSeeder.php:156]: move it @spektra2147

File: addons/default/visiosoft/advs-module/src/Notifications/AdvsModuleNotificationsTemplateSeeder.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        'greeting' => 'Hi',
+                        'subject' => 'Pending Ad'
+                    ],
+                    'stream' => 'advs',
+                    'slug' => Str::slug('Pending Ad', '_')
+                ]
+            ];
+
+            foreach ($templates as $template) {
+                $this->createTemplate($template);
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/src/Notifications/AdvsModuleNotificationsTemplateSeeder.php:68]: dispatch

**Review by diashalabi — CHANGES_REQUESTED** [review]

**Review by diashalabi — APPROVED** [review]

**Pull request merged**